package.json

```diff
@@ -18,6 +18,7 @@
     "babelify": "^7.2.0",
     "deepcopy": "^0.5.0",
```

Comment:
`deepcopy` can be done with `object-extend`, but writing deepcopy does a better job at conveying intent. It is also a very small package.